export const global = {
   	catchError: (error) => {
    	console.log(error)
    }
}